doc/tokens/tokentypes/yubikey.rst

```diff
@@ -3,22 +3,33 @@
 Yubikey
 -------
 
-.. index:: Yubikey, Yubico AES mode
+.. index:: Yubikey, Yubico AES mode, Yubikey OATH-HOTP mode
 
-The Yubikey is initialized with privacyIDEA and works in Yubicos own AES mode.
-It outputs a 44 character OTP value, consisting of a 12 character prefix and
+The Yubikey can be used with privacyIDEA in Yubico's own AES mode (*Yubico OTP*)
+or in the OATH-HOTP mode which is for privacyIDEA a HOTP Token.
+
+In AES mode it outputs a 44 character OTP value, consisting of a 12 character prefix and
 a 32 character OTP. But in contrast to the :ref:`yubico_token` Cloud
 mode, in this mode the secret key is contained within the token and your own
-privacyIDEA installation.
+privacyIDEA installation. In OATH-HOTP mode, it just prints a series of 6 or 8 digits.
```

Comment:
The problem is, that the OATH-HOTP mode does not belong here.
Since this section deals about the tokentype "yubikey". Which **is** the the AES-mode.

Note: This section here does not deal with *all* possible yubikey modes, so it should probably refer for this to other sections.

We probably should restructure the lines starting with line 22.